### PR TITLE
3.20.8

### PR DESCRIPTION
**Pull request by jorgegutierrez2019**
**Added:**

personal-homepage

G3458397_Personal_Homepage_Meridian_Icon

### CODE REVIEW

File: docs/keywords.json [hunk 1]
```diff
@@ -11098,5 +11098,20 @@
   ],
   "category":"GIS",
   "release":"3.20.7"
+}, 
+"personal-homepage":{
+  "alias":[
+    " 3.20.8",
```
Comment (paulcpederson) [inline docs/keywords.json:11104]: extra space here

**Comment by paulcpederson** [comment]
little nitpick but looks good

**Comment by jorgegutierrez2019** [comment]
Thanks Paul, how do I fix this? I’m not too familiar with this process.



From: Paul Pederson ***@***.***>
Reply-To: Esri/calcite-ui-icons ***@***.***>
Date: Wednesday, December 21, 2022 at 2:10 PM
To: Esri/calcite-ui-icons ***@***.***>
Cc: Jorge Gutierrez ***@***.***>, Author ***@***.***>
Subject: Re: [Esri/calcite-ui-icons] 3.20.8 (PR #248)


little nitpick but looks good

**Comment by jorgegutierrez2019** [comment]
Nevermind, I think I got it.

thanks

From: Paul Pederson ***@***.***>
Reply-To: Esri/calcite-ui-icons ***@***.***>
Date: Wednesday, December 21, 2022 at 2:10 PM
To: Esri/calcite-ui-icons ***@***.***>
Cc: Jorge Gutierrez ***@***.***>, Author ***@***.***>
Subject: Re: [Esri/calcite-ui-icons] 3.20.8 (PR #248)


little nitpick but looks good

**Review by nicklu10 — APPROVED** [review]
looks good to me

**Pull request merged**